driftbase/api/clients.py

```diff
@@ -342,6 +347,8 @@ def delete(self, client_id):
 
         current_app.extensions["messagebus"].publish_message("client", message_data)
 
+        richpresence.set_online_status(player_id, False)
```

Comment:
You want to ensure this never throws, or wrap it and ignore failures, since this is a side effect and not a requirement for the success of DELETE /clients/n